README.md

```diff
@@ -86,6 +88,9 @@ An optional `--eigenda-eth-confirmation-depth` flag can be provided to specify a
 
 An ephemeral memory store backend can be used for faster feedback testing when testing rollup integrations. To target this feature, use the CLI flags `--memstore.enabled`, `--memstore.expiration`.
 
+### S3 Fallback
+An optional S3 fallback `--s3.backup` can be triggered to ensure resiliency when **reading**. When enabled, a blob is persisted to S3 after being successfully dispersed using the keccak256 hash of the existing commitment for the entity key. In the event that blobs cannot be read from EigenDA, they will then be retrieved from S3. 
```

Comment:
Dont understand what this means.
Where does "existing commitment" come from?
What does "entity key" mean? Not mentioned anywhere else in the README.

Reply:
just S3 key and the commitment generated by eigenda